Makefile

```
@@ -14,6 +14,8 @@ MIN_MAP = $(DIST)/react-grid-layout.min.js.map
 
 build: clean build-js $(MIN)
 
+build-dev: clean build-js-dev $(MIN)
```

checkmake (0.3.2)

[warning] Target "build-dev" should be declared PHONY.

(phonydeclared)